src/experiments/preferences/schema.json

```diff
@@ -97,6 +97,24 @@
         ],
         "async": true
       },
+      {
+        "name": "setIntPref",
+        "type": "function",
+        "description": "Sets the value of a integer preference",
+        "parameters": [
+          {
+            "type": "string",
+            "name": {
```

Comment:
All of these enums should be defined at the same level as the `type`. Not nested under `name`.